events/wc_titanforged_events.txt

```diff
@@ -1 +1,218 @@
+				}
+			}
+		}
+		character_event = { id = WCUTF.3 days = 7 }
+		
+		ai_chance = {
+			factor = 0
+		}
+	}
+}
+character_event = {
+	#Basic event information
+	id = WCUTF.2
+	desc = {
+		trigger = {
+			character = 400501
+		}
+		text = EVTDESC_WCUTF.2_loken
+	}
+
+	desc = {
+		trigger = {
+			character = 400502
+		}
+		text = EVTDESC_WCUTF.2_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+
+	immediate = {
+	}
+	option = {
+		name = EVTOPTA_WCUTF.2
+		trigger = {
+			character = 400501
+		}
+		religion = old_gods_worship
+	}
+	option = {
+		name = EVTOPTB_WCUTF.2
+		trigger = {
+			character = 400502
+		}
+		religion = old_gods_worship
+		grant_title_no_opinion  = d_ulduar
+		grant_title_no_opinion  = c_ulduar
+		grant_title_no_opinion  = c_eanorah
+		c_400501 = {
+			death = { death_reason = death_duel killer = PREV }
+		}
+	}
+	after = {
+		d_ulduar = { any_de_jure_vassal_title = { location = { remove_province_modifier = ulduar_closed_down } } }
+		set_global_flag = ulduar_opened_flag
+	}
+}
+character_event = {
+	#Basic event information
+	id = WCUTF.3
+	desc = {
+		trigger = {
+			character = 400501
+		}
+		text = EVTDESC_WCUTF.3_loken
+	}
+
+	desc = {
+		trigger = {
+			character = 400502
+		}
+		text = EVTDESC_WCUTF.3_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+
+	immediate = {
+	}
+	option = {
+		name = EVTOPTA_WCUTF.3
+		trigger = {
+			character = 400501
+		}
+		set_player_character = event_target:target_thorim
+		event_target:target_thorim = {
+			grant_title_no_opinion  = d_ulduar
+			grant_title_no_opinion  = c_ulduar
+			grant_title_no_opinion  = c_eanorah
+		}
+		c_400501 = {
+			death = { death_reason = death_suicide }
+		}
+	}
+	option = {
+		name = EVTOPTB_WCUTF.3
+		trigger = {
+			character = 400502
+		}
+		grant_title_no_opinion  = d_ulduar
+		grant_title_no_opinion  = c_ulduar
+		grant_title_no_opinion  = c_eanorah
+		religion = mystery_of_the_makers
+		c_400501 = {
+			death = { death_reason = death_duel killer = PREV }
+		}
+	}
+	after = {
+		event_target:target_thorim = {
+			d_ulduar = {
+				any_de_jure_vassal_title = {
+					location = {
+						remove_province_modifier = ulduar_closed_down
+						religion = mystery_of_the_makers
+					}
+				}
+			}
+			any_realm_character = {
+				limit = { religion = cult_of_loken }
+				religion = mystery_of_the_makers
+			}
+			set_global_flag = ulduar_opened_flag
+			character_event = { id = WCUTF.4 days = 1}
+		}
+	}
+}
+character_event = {
+	#Basic event information
+	id = WCUTF.4
+	hide_window = yes
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+
+	immediate = {
+		any_realm_character = {
+			limit = { religion = cult_of_loken }
+			religion = mystery_of_the_makers
+		}
+	}
+}
```

Comment:
What is the point of the event?

Reply:
religion change doesnt work without it for some chars

Reply:
Which ones?

Reply:
Lokens vassals

Reply:
@sejtalk, so I delete this event and can't see Loken religion in my court.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/78928770-3cbce080-7ab2-11ea-8915-0dcb21fe9822.png)

</details>